tests/unit_tests/cpu/tools_helix_trajectory.cpp

```diff
@@ -193,3 +193,29 @@ GTEST_TEST(detray_intersection, helix_trajectory_small_pT) {
     EXPECT_NEAR(true_pos[1], helix_pos[1], tol);
     EXPECT_NEAR(true_pos[2], helix_pos[2], tol);
 }
+
+GTEST_TEST(helix, direction_stability) {
+
+    using vector3 = test::vector3;
+    using point3 = test::point3;
+    using transform3_type = test::transform3;
+
+    // magnetic field
+    const vector3 B{0.f, 0.f, 1.f * unit<scalar>::T};
+
+    const point3 pos{0.f, 0.f, 0.f};
+    const scalar time{0.f};
+    const vector3 mom{1.f, 1.f, 1.f * unit<scalar>::GeV};
```

Comment:
The first two elements are in MeV right now, lets make that explicit
```suggestion
    const vector3 mom{1.f * unit<scalar>::MeV, 1.f * unit<scalar>::MeV, 1.f * unit<scalar>::GeV};
```

Reply:
Nvm, I got confused.